core-web/libs/sdk/react/src/lib/next/types.ts

```diff
@@ -0,0 +1,442 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Are this type internal only? are some of this goes to `/client/type` at some point?